### ISSUE TITLE
Use twilio dev-phone on ec2

### ISSUE THREAD

**Issue by snowmous3** [twilio-labs/dev-phone#185]
Hi, I started an ec2 instance with the dev-phone on there.
However, I had a problem with the service when the application started, and I tried to do a call. The call can't start, and I can't listen to the call.
Another weird thing is I can't log any error or something.
Do I need to open another specific port? In addition to 1337.

**Comment by ayyrickay** [comment]
Hey @snowmous3, this isn't currently a supported use case for the Dev Phone, so I'm honestly not sure. This is something that we could look into at a later date, especially when we think about ways to deploy Dev Phones in different places. At the moment, however, we need to prioritize some other tasks.

For what it's worth, the dev phone works by deploying a server from the command line. For EC2, this is probably overkill, and might be the source of your issues - I wonder if some Dev Phone resources aren't getting called properly because of the strange server spin up. Do you see any errors in your browser DevTools?

**Comment by babay123** [comment]
If you want to use the dev-phone for calls on a remote server, you need to set up an HTTPS server. This is because the call functionality utilizes the `getUserMedia` function, which requires either HTTPS or localhost to work properly.

Alternatively, you can add your EC2 instance's address to your Chrome whitelist. To do this, navigate to `chrome://flags/#unsafely-treat-insecure-origin-as-secure` and add your EC2 address.

**Comment by snowmous3** [comment]
> Hey @snowmous3, this isn't currently a supported use case for the Dev Phone, so I'm honestly not sure. This is something that we could look into at a later date, especially when we think about ways to deploy Dev Phones in different places. At the moment, however, we need to prioritize some other tasks.
> 
> For what it's worth, the dev phone works by deploying a server from the command line. For EC2, this is probably overkill, and might be the source of your issues - I wonder if some Dev Phone resources aren't getting called properly because of the strange server spin up. Do you see any errors in your browser DevTools?

No, I can't see any error. But I imagine could be something about the AWS. Like any role or something.
